wolfssl/test.h

```diff
@@ -155,6 +155,7 @@
     #pragma warning(disable:4244 4996)
 #endif
 
+#define WOLFSSL_CIPHER_LIST_MAX_SIZE 4096
```

Comment:
Please make this overridable `#ifndef WOLFSSL_CIPHER_LIST_MAX_SIZE`. Also did you check how big the maximum actually is... if all ciphers are enabled (STATIC_RSA, STATIC_ECC, PSK, CCM, etc...)? We should make sure this is actually big enough or not "too" large.